guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+[id="enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+The installation of {Project} does not have any Puppet integration configured by default.
+Users will need enable the approprite integration for their situation.
+{Project} can be configured to manage and deploy Puppet Server on the {ProjectServer} or on a {SmartProxy}.
+Additionally, Puppet Server can be deployed externally to {Project} and integrated for reporting, facts and ENC.
+
+.Prerequisites
+
+* Puppet integration must be first enabled on {Project} before being enabled on any {SmartProxy}.
+
+.Procedure
+
+. On {Project}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-puppet --enable-foreman-cli-puppet --foreman-proxy-puppet true --foreman-proxy-puppetca true --foreman-proxy-content-puppet true --enable-puppet --puppet-server true --puppet-server-foreman-ssl-ca /etc/pki/katello/puppet/puppet_client_ca.crt --puppet-server-foreman-ssl-cert /etc/pki/katello/puppet/puppet_client.crt --puppet-server-foreman-ssl-key /etc/pki/katello/puppet/puppet_client.key
```

Comment:
In other places long lines are split over multiple lines.

Reply:
Will take care of it.